### PR TITLE
Add step-kms-plugin attest command

### PR DESCRIPTION
**Pull request by maraino**
### Description

This PR adds `step-kms-plugin attest` command.

### CODE REVIEW

File: cmd/attest.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+			URI: kuri,
+		})
+		if err != nil {
+			return fmt.Errorf("failed to load key manager: %w", err)
+		}
+		defer km.Close()
+
+		attester, ok := km.(apiv1.Attester)
+		if !ok {
+			return fmt.Errorf("%s does not implement an Attester", kuri)
```
Comment (hslatman) [inline cmd/attest.go:59]:
```suggestion
			return fmt.Errorf("%s does not implement Attester", kuri)
```

File: cmd/attest.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+			}
+			return nil
+		case resp.PublicKey != nil:
+			block, err := pemutil.Serialize(resp.PublicKey)
+			if err != nil {
+				return err
+			}
+			return pem.Encode(os.Stdout, block)
+		default:
+			return fmt.Errorf("failed to create attestation: unsupported response")
```
Comment (hslatman) [inline cmd/attest.go:93]:
```suggestion
			return errors.New("failed to create attestation: unsupported response")
```

File: cmd/attest.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+	"github.com/smallstep/step-kms-plugin/internal/flagutil"
+	"github.com/spf13/cobra"
+	"go.step.sm/crypto/kms"
+	"go.step.sm/crypto/kms/apiv1"
+	"go.step.sm/crypto/pemutil"
+)
+
+// attestCmd represents the attest command
+var attestCmd = &cobra.Command{
```
Comment (hslatman) [inline cmd/attest.go:30]: We probably need to think about the use cases and grouping multiple commands in an `attest(ation)` namespace (or something like that) going forward. In this command the (embedded) attestation certificate is returned, but we'll probably need commands for creating attested keys in a YubiKey in the future too, for example. I think there's value in providing high level, simple invocations that abstract the complexity of interacting with the hardware components.

**Review by hslatman — APPROVED** [review]

File: cmd/attest.go (hunk 3 above)
Comment (maraino) [reply cmd/attest.go:31]: On a YubiKey or any other KMS, you can create `step-kms-plugin create`, and you can import or print a certificate with `step-kms-plugin certificate`. So for those specific cases, you are already covered. At the moment, the only case that comes to mind is an action that combines two commands. For example, creating an attestation certificate and using the same key to sign a challenge. Right now, I'm doing this in the cli as two different commands.

Comment (maraino) [reply cmd/attest.go:31]: I'll merge this as it is, but won't create a new release until we figure out if we need to change this. Let's talk.

**Pull request merged**